test/e2e-sno-disruptive/sno_disruptive_test.go

```diff
@@ -23,62 +23,82 @@ import (
 func TestFallback(tt *testing.T) {
 	t := commontesthelpers.NewE(tt)
 	cs := getClients(t)
+	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Minute)
```

Comment:
just use `context.TODO()` the test will be terminated by the framework if it take too long to execute.

Reply:
Updated.